### DISCUSSION TITLE
Wifi Proxy

### DISCUSSION THREAD

**Discussion by seeddddd** [3kh0/ext-remover#566] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is there a way to disable the proxy on my school wifi?

**Comment by BroJac5246** [comment, marked as answer]
If it's happening on the router's end and not on-device, then no, unless you fancy hacking the Wi-Fi network. In other words, you can't. However, you could connect to a hotspot, which wouldn't have the proxy. From there, something like New Point-Blank could be used to access anything.
